components/grid.css

```diff
@@ -0,0 +1,154 @@
+/* grid */
+
+/* 2 col grid */
+
+/* 50 50 grid */
+
+.grid-2 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+.grid-2 > :nth-child(1) {
+  grid-column: 1/2;
```

Comment:
You don't need to specify the grid column for direct children as they automatically get that space.

Reply:
Got it , Thanks